alacritty_terminal/src/term/mod.rs

```diff
@@ -1197,7 +1195,10 @@ impl<T> Term<T> {
         }
 
         self.alt = !self.alt;
-        mem::swap(&mut self.grid, &mut self.alt_grid);
+        mem::swap(&mut self.grid, &mut self.inactive_grid);
+        mem::swap(&mut self.cursor, &mut self.inactive_cursor);
+        self.cursor.point.line = min(self.cursor.point.line, self.grid.num_lines() - 1);
+        self.cursor.point.col = min(self.cursor.point.col, self.grid.num_cols() - 1);
```

Comment:
What happens when this is removed? I don't think it should be necessary, since we do reflow saved cursors.

Reply:
Tests still pass if this is removed